packages/fast-usdc/src/fast-usdc.contract.ts

```diff
@@ -102,10 +101,9 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
      *
      * Provide an API call in the form of an invitation maker, so that the
      * capability is available in the smart-wallet bridge.
-     *
-     * @param {CctpTxEvidence} evidence
+     * @param evidence
```

Comment:
Is this `@param` still necessary?
Perhaps only until we tell eslint otherwise?

Reply:
Right. Our eslint config is that if there's a jsdoc comment on a function it must have params. I'd like to leave changing out that as out of scope of this PR.

Reply:
It’s not awful to have a place to document the meaning of the param for tsdoc. We just don’t have to note the type.